Refactor PowerShell install module (#1473)
open-telemetry/opentelemetry-dotnet-instrumentation: docs/README.md

# Setup your Windows Service instrumentation
Register-OpenTelemetryForWindowsService -WindowsServiceName "MyServiceName" -OTelServiceName "MyServiceDisplayName"

# Setup environment to start instrumentation from the current PowerShell session
Register-OpenTelemetryForCurrentSession -OTelServiceName "MyServiceDisplayName"

# Get current installation location
Get-OpenTelemetryInstallDirectory

# List all available commands
Get-Command -Module install

# Get command's usage information
Get-Help Install-OpenTelemetryCore -Detailed
```

> ⚠️ You have to restart the IIS / Windows service

Reviewer: Maybe it could add the commands to the snippet as well?

```
Restart-Service -Name "MyServiceName"
& iisreset
```

Author: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1473/commits/fe06b8f6dfbd72d3186e5bb138e7e7a8fbe44bce

Author: Warning also had to be changed >  https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1473/commits/9c78a6ff0fec6429f93b555d493771b5a3b42bec

# Setup your Windows Service instrumentation
Register-OpenTelemetryForWindowsService -WindowsServiceName "MyServiceName" -OTelServiceName "MyServiceDisplayName"

# Setup environment to start instrumentation from the current PowerShell session
Register-OpenTelemetryForCurrentSession -OTelServiceName "MyServiceDisplayName"

# Get current installation location
Get-OpenTelemetryInstallDirectory

# List all available commands
Get-Command -Module install

# Get command's usage information
Get-Help Install-OpenTelemetryCore -Detailed
```

> ⚠️ Registering for the IIS or a Windows service will perform a service restart!

Reviewer: ```suggestion
⚠️ Registering for IIS and Windows Service will perform a service restart.
```
